Formula/ciphey.rb

```diff
@@ -0,0 +1,191 @@
+    sha256 "c389c1d06bf7904078ca03399a4816f974a1d590090fecea0c63ec26ebaf1cef"
+  end
+
+  resource "pyflakes" do
+    url "https://files.pythonhosted.org/packages/a8/0f/0dc480da9162749bf629dca76570972dd9cce5bedc60196a3c912875c87d/pyflakes-2.3.1.tar.gz"
+    sha256 "f5bc8ecabc05bb9d291eb5203d6810b49040f6ff446a756326104746cc00c1db"
+  end
+
+  resource "Pygments" do
+    url "https://files.pythonhosted.org/packages/ba/6e/7a7c13c21d8a4a7f82ccbfe257a045890d4dbf18c023f985f565f97393e3/Pygments-2.9.0.tar.gz"
+    sha256 "a18f47b506a429f6f4b9df81bb02beab9ca21d0a5fee38ed15aef65f0545519f"
+  end
+
+  resource "pylint" do
+    url "https://files.pythonhosted.org/packages/59/9a/3a021b3b7965c5070bfec5d54759f076a31bd537043e0dc9b0fb2b49bff6/pylint-2.9.3.tar.gz"
+    sha256 "23a1dc8b30459d78e9ff25942c61bb936108ccbe29dd9e71c01dc8274961709a"
+  end
+
+  resource "pywhat" do
+    url "https://files.pythonhosted.org/packages/32/18/94ed2965c98f2577826bf642bd496516738c3056c824687c1453c2e88944/pywhat-1.1.0.tar.gz"
+    sha256 "445cfe9ac2ccffd8438d4d4197fc5ec0ebbfac1ec241a75cd2e65ea5ed68e615"
+  end
+
+  resource "PyYAML" do
+    url "https://files.pythonhosted.org/packages/a0/a4/d63f2d7597e1a4b55aa3b4d6c5b029991d3b824b5bd331af8d4ab1ed687d/PyYAML-5.4.1.tar.gz"
+    sha256 "607774cbba28732bfa802b54baa7484215f530991055bb562efbed5b2f20a45e"
+  end
+
+  resource "rich" do
+    url "https://files.pythonhosted.org/packages/2f/b7/b8b950458f88c6d74978a37ad1d1fb9885464372fcb3d4077c2d9186a5c3/rich-10.4.0.tar.gz"
+    sha256 "6e8a3e2c61e6cf6193bfcffbb89865a0973af7779d3ead913fdbbbc33f457c2c"
+  end
+
+  resource "toml" do
+    url "https://files.pythonhosted.org/packages/be/ba/1f744cdc819428fc6b5084ec34d9b30660f6f9daaf70eead706e3203ec3c/toml-0.10.2.tar.gz"
+    sha256 "b3bda1d108d5dd99f4a20d24d9c348e91c4db7ab1b749200bded2f839ccbe68f"
+  end
+
+  resource "wrapt" do
+    url "https://files.pythonhosted.org/packages/82/f7/e43cefbe88c5fd371f4cf0cf5eb3feccd07515af9fd6cf7dbf1d1793a797/wrapt-1.12.1.tar.gz"
+    sha256 "b62ffa81fb85f4332a4f609cab4ac40709470da05643a082ec1eb88e6d9b97d7"
+  end
+
+  def install
+    venv = virtualenv_create(libexec, Formula["python@3.9"].opt_bin/"python3")
+    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+
+    resource("cipheycore").stage do
+      args = std_cmake_args + %W[
+        -DCIPHEY_CORE_TEST=OFF
+        -DCIPHEY_CORE_PYTHON=#{Formula["python@3.9"].opt_frameworks}/Python.framework/Versions/#{xy}
+      ]
+      system "cmake", "-S", ".", "-B", "build", *args
+      system "cmake", "--build", "build", "-t", "ciphey_core"
+      system "cmake", "--build", "build", "-t", "ciphey_core_py", "--config", "Release"
+
+      cd "build" do
+        system "poetry", "build", "-f", "sdist", "--verbose"
+        venv.pip_install Pathname(Dir[Pathname.pwd/"dist/cipheycore-*"].first)
```

Comment:
```suggestion
        venv.pip_install Dir["dist/cipheycore-*"].first
```
Doesn't this work?

Reply:
Initially I had concerns about `venv.pip_install` treating strings differently but this seems to work.

Reply:
They should be treated the same way -- neither of them `repond_to?(:stage)`.

Reply:
I recall there was a mode where you can pass a string containing the contents of a `requirements.txt` file but I believe the string has to be multiline for that to happen.

In any case, I find this kind of method overloading to be a bit of a weird code smell when I have to keep track of these kinds of corner cases.